en/core-libraries/form.rst

```diff
@@ -98,6 +98,42 @@ accordingly. We could have also used the ``validate()`` method to only validate
 the request data::
 
     $isValid = $form->validate($this->request->data);
+    
+Setting Form Values
+===================
+
+In order to set the values for the fields of a modelless form, you can define
```

Comment:
If that whole section is about that sure, it just sounded like this was only true for those :)